app/controllers/catalog_controller.rb

```diff
@@ -478,7 +478,6 @@ class CatalogController < ApplicationController
     #   field.include_in_simple_select = false
 
     config.add_search_field 'all_fields', label: 'Keyword' do |field|
```

Comment:
Does this still need to take a block, without the conditional? Or could it just be:
```
    config.add_search_field 'all_fields', label: 'Keyword'
```

(can't seem to do this as a "suggestion" since it has deleted lines or something)